pygfunction/gfunction.py

```diff
@@ -1236,11 +1241,11 @@ def _format_inputs(self, boreholes_or_network):
 
         """
         # Convert borehole to a list if a single borehole is provided
-        if isinstance(boreholes_or_network, Borehole):
+        if boreholes_or_network.ghe_type == GHEType.BOREHOLE:
             boreholes_or_network = [boreholes_or_network]
         # Check if a borefield or a network is provided as an input and
         # correctly assign the variables self.boreholes and self.network
-        if isinstance(boreholes_or_network, Network):
+        elif boreholes_or_network.ghe_type == GHEType.NETWORK:
```

Comment:
It seemed to me like these should be chained if-else statements, not a series of `if` blocks, since they didn't `return` early.